src/www/_partials/component-example/_component-example.scss

```diff
@@ -29,7 +29,7 @@
 }
 
 span[class^="hljs-"] {
-  background: #000000;
+  background: $black;
```

Comment:
Make sure that the background colour isn't black. It should be the same as the background colour of the hljs background. Otherwise you get  a weird effect like:
![image](https://user-images.githubusercontent.com/13015201/90523279-97eb9780-e164-11ea-8d0e-0dca16032f29.png)

```suggestion
  background: #2b2b2b;
```

The only reason we are doing this is because AXE can't determine the background-colour properly.